### PR TITLE
[shell] fix volume grow in shell

### PR DESCRIPTION
**Pull request by kmlebedev**
# What problem are we solving?

force volume grow via shell

# How are we solving the problem?

add VolumeGrow grpc func 

# How is the PR tested?

localy
```
make server
> volume.grow -collection test -dataCenter DefaultDataCenter -rack DefaultRack -dataNode 192.168.3.43:8080 -count 1
```

# Checks
- [ ] I have added unit tests if possible.
- [ ] I will add related wiki document changes and link to this PR after merging.

### CODE REVIEW

**Comment by chrislusf** [comment]
why a new volume grow grpc API is needed?

**Comment by kmlebedev** [comment]
> why a new volume grow grpc API is needed?

Because the Assign function is created for something else and it is limited to checking shouldGrow, and I want to have grpc logic similar to http volumeGrowHandler

File: weed/pb/master.proto [hunk 1]
```diff
@@ -51,6 +51,8 @@ service Seaweed {
   }
   rpc RaftRemoveServer (RaftRemoveServerRequest) returns (RaftRemoveServerResponse) {
   }
+  rpc VolumeGrow (AssignRequest) returns (VolumeGrowResponse) {
```
Comment (chrislusf) [inline weed/pb/master.proto]: create a VolumeGrowRequest and remove unnecessary fields

Comment (kmlebedev) [reply weed/pb/master.proto]: done

**Pull request merged**